spec/validators/chat_allow_uploads_validator_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe ChatAllowUploadsValidator do
```

Comment:
```suggestion
RSpec.describe ChatAllowUploadsValidator do
```

Just a minor note but in RSpec 4, we need to use `RSpec.describe` as `describe` will be deprecated.